smithy-model/src/test/java/software/amazon/smithy/model/validation/testrunner/SmithyTestCaseTest.java

```diff
@@ -32,14 +33,17 @@
 public class SmithyTestCaseTest {
     @Test
     public void validatesThatEventsAreValid() {
-        Assertions.assertThrows(
+        IllegalArgumentException e = Assertions.assertThrows(
                 IllegalArgumentException.class,
-                () -> SmithyTestCase.parseValidationEvent("[ERROR] - m"));
+                () -> SmithyTestCase.parseValidationEvent("[ERROR] - m", "filename"));
+
+        assertTrue(e.getMessage().contains("`filename`"));
+        assertTrue(e.getMessage().contains("SUPPRESSED|NOTE|WARNING|DANGER|ERROR"));
```

Comment:
I'm not sure I understand this bit, is the exception message expected to contain the literal string `"SUPPRESSED|NOTE|WARNING|DANGER|ERROR"`?